.github/workflows/run-test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
   tests:
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
Had to remove `3.7` because in GHA, this array can only hold 5 items. And running tests in python v3.12 is preferred to running them in v3.7.